cli/src/opts/cast.rs

```diff
@@ -147,6 +147,12 @@ pub enum Subcommands {
         sig: String,
         #[clap(help = "the list of arguments you want to call the function with")]
         args: Vec<String>,
+        #[clap(long, help = "gas quantity for the transaction")]
+        gas: Option<U256>,
+        #[clap(long, help = "ether value for the transaction")]
```

Comment:
needs more clarification regarding unit

Reply:
would - **ether value (in wei) for the transaction** make it clear?